gnosis/conditional-markets-interface
# Feature/#91 add view for sumsub sdk

# Description
This PR add tier 2 flow after receiving the email when creating an account.

# Code
This PR will create a new Tier 2 view (it was requested explicitly not to be modals). Although it will follow same step scheme defined for Tier modals.

# To Test
- Go to any Sight market with an unregistered Ethereum account and create a new account.
  - Make sure to use a legit email, as this process will sent unique codes there.
  - This should work also going first through Tier 1 but can be directly checked asking for Tier 2
- Email link could not be 100% functional (route could not be final yet). 
  - Open the link and copy the query parameters. (sdk_client, sdk_token, email)
  - Open the test environment in route `/#/kyc/tier2` and add the query params
  - Reload the page
- Follow the steps until finally verified
  - Final verification must be done manually in SumSub test aplication.
